data-collections-exercises.md

```diff
@@ -15,83 +15,91 @@ Access the second value in `planeteers`.
 Add "Heart" to the end of `planeteers`.
 
 ```rb
-# Your answer here
+planeteers.push("Heart")
 ```
 
 Remove "Captain Planet" from the array (without using a method).
 
 ```rb
-# Your answer here
+planeteers.delete("Captain Planet")
 ```
 
 Combine `planeteers` with `rangers = ["Red", "Blue", "Pink", "Yellow", "Black"]` and save the result in a variable called `heroes`.
 
 ```rb
-# Your answer here
+heroes = rangers.combination(planeteers)
 ```
 
 Alphabetize the contents of `heroes` using a method. [The Ruby documentation might help](http://ruby-doc.org/core-2.6.1/Array.html).
 
 ```rb
-# Your answer here
+heroes.sort
 ```
 
 Randomize the contents of `heroes` using a method. [The Ruby documentation might help](http://ruby-doc.org/core-2.6.1/Array.html).
 
 ```rb
-# Your answer here
+heroes.shuffle
 ```
 
 #### Bonus
 
 Select a random element from `heroes` using a method. [The Ruby documentation might help](http://ruby-doc.org/core-2.6.1/Array.html).
 
 ```rb
-# Your answer here
+ heroes[rand(myarray.length)]
 ```
 
 Select all elements in `heroes` that begin with "B" using a method. [The Ruby documentation might help](http://ruby-doc.org/core-2.6.1/Array.html).
 
 ```rb
-# Your answer here
+heroes[2..-1]
```

Comment:
heroes[2..-1] will return the third element of the array to solve the problem you need to iterate through the heroes array and check for the first letter if it's `B` or not 

```
heroes.select{|heros| heros[0] === 'B'}
```